controllers/grid/users/chapter/form/ChapterForm.inc.php

```diff
@@ -90,17 +90,23 @@ function initData() {
 
 		$monograph = $this->getMonograph();
 		$this->setData('submissionId', $monograph->getId());
+		$chapterPublicationDatesEnabledSubmission = (int)$monograph->getEnableChapterPublicationDates() === 1;
+		$this->setData('chapterPublicationDatesEnabledSubmission', $chapterPublicationDatesEnabledSubmission);
```

Comment:
Let's use the same symbolic key in the form data as we use in the monograph data. So change `chapterPublicationDatesEnabledSubmission` to `enableChapterPublicationDates`.

Reply:
@NateWr  sure, changed it  with https://github.com/pkp/omp/pull/687/commits/43f2e36adad93c38b7b6bd6c32f65e56e0c27985